src/io/flutter/survey/FlutterSurvey.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.survey;
+
+import com.google.gson.JsonObject;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class FlutterSurvey {
+  final String uniqueId;
+  final LocalDate startDate;
+  final LocalDate endDate;
+
+  final String title;
+  final String urlPrefix;
+
+  FlutterSurvey(String uniqueId, String startDate, String endDate, String title, String urlPrefix) {
+    this.uniqueId = uniqueId;
+    this.startDate = parseDate(startDate);
+    this.endDate = parseDate(endDate);
+    this.title = title;
+    this.urlPrefix = urlPrefix;
+  }
+
+  private static LocalDate parseDate(String date) {
+    return LocalDate.from(DateTimeFormatter.ISO_DATE_TIME.parse(date));
+  }
+
+  public static FlutterSurvey fromJson(@NotNull JsonObject json) {
+    return new FlutterSurvey(json.getAsJsonPrimitive("uniqueId").getAsString(),
+                             json.getAsJsonPrimitive("startDate").getAsString(),
+                             json.getAsJsonPrimitive("endDate").getAsString(),
+                             json.getAsJsonPrimitive("title").getAsString(),
+                             json.getAsJsonPrimitive("url").getAsString()
+    );
+  }
+
+  boolean isSurveyOpen() {
+    final LocalDate now = LocalDate.now();
+    return now.isAfter(startDate) && now.isBefore(endDate);
```

Comment:
I assume that the start date is inclusive, and the end date is exclusive? Or are they both inclusive?

This should likely read something like `now.isEqualToOrAfter(startDate)`.

Reply:
I guess inclusive/exclusive won't matter that much, because both start date and end date actually specify the time, in addition to the date. So something consistent would work..?

```
"startDate": "2020-03-03T09:00:00-08:00",
"endDate": "2020-03-10T09:00:00-07:00"
```

Or perhaps I should change the names of the objects to `startDateAndTime` and 'endDateAndTime` to avoid confusion?

Reply:
Ah, I'd written that thinking that it was for a day, not a date+time; agree that it doesn't matter if we're just talking about a second or millisecond.

Reply:
>  if we're just talking about a second or millisecond

Exactly my thinking.

> Or perhaps I should change the names of the objects to startDateAndTime and 'endDateAndTime` to avoid confusion?

I don't  think it's necessary but  happy to go with the flow.

Reply:
Updated to `now.isEqual(startDate) || now.isAfter(startDate)` ...